pasqal_cloud/__init__.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2020 Pasqal Cloud Services development team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from typing import Any, Dict, List, Optional
```

Comment:
We seem to be inconsistent with the typehints in the SDK, we are using capital case typehints and lower case. Could we use `from __futures__ import annotations` and swap the typehinting to lower case for consistency? I understand you might not have added this, but it's a nice chance to take care of it now

For example `[List[int]` would become `list[int]`
I do not think there's a technical reason to prevent this but let me know if there is